# Serial params
serial:
  port: "/dev/ttyUSB0"
  baudrate: 230400
  rate: 10

# Classification params
model: 
  dataroot: "/home/frankie/catkin_ws/src/dataglove/classification/dataset"
  resultroot: "/home/frankie/catkin_ws/src/dataglove/classification/results"
  resultsuffix: ""
  n_inp: 21
  n_hid: 256
  batch: 128
  dt: 0.042
  gamma: 2.7
  epsilon: 4.7
  gamma_range: 2.5
  epsilon_range: 4.5
  device: "cpu"  # choices: ["cpu", "cuda"]
  input_scaling: 1.0
  rho: 0.99
  leaky: 1.0
  use_test: true
  trials: 1
  topology: "full"  # choices: ["full", "ring", "band", "lower", "toeplitz", "orthogonal"]
  sparsity: 0.0
  reservoir_scaler: 1.0
  threshold: 0.5
  rc: 100.0
  reset: 0.01
  bias: 0.01
  #mixron
  perc: 0.5
  #lsm
  win_e: 1
  win_i: 0.5
  w_e: 1
  w_i: 0.5
  Ne: 200
  Ni: 56
  #model choice
  sron: false 
  mixron: false 
  liquidron: true

#labels
rigidity: {'hard': 1, 'soft': 0}
obj: {'ball': 1, 'bottle': 0}

# start_sron: false
# start_mixron: false
# start_lsm: false

buffer_size: 100
